utility/src/main/java/org/oppia/util/parser/ImageLoaderModule.kt

```diff
@@ -0,0 +1,17 @@
+package org.oppia.util.parser
+
+import dagger.Module
+import dagger.Provides
+import javax.inject.Singleton
+
+/** Provides image loading dependencies. */
+@Module
+class ImageLoaderModule {
+
+  @Provides
+  @Singleton
+  @ImageLoaderAnnotation
+  fun providesGlideImageLoader(): ImageLoader {
```

Comment:
Prefer using ``Binds`` since then you can mark ``GlideImageLoader``'s constructor as ``@Inject``:

```kotlin
@Module
abstract class ImageLoaderModule {
  @Binds
  abstract fun provideGlideImageLoader(impl: GlideImageLoader): ImageLoader
}
```
Note I also removed the ``@Singleton`` scope. We don't need to scope this object because it doesn't contain state, so it's fine if Dagger recreates it each time we inject it.

Reply:
I tried using @Binds and used above implementation. Its throwing me error
`error: cannot access SimpleTarget class file for com.bumptech.glide.request.target.SimpleTarget not found`

Reply:
This is removed and resolved error.